webpack/webpack.config.common.js

```diff
@@ -0,0 +1,46 @@
+const path = require('path')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin')
+const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin')
+const Dotenv = require('dotenv-webpack')
+
+module.exports = {
+	entry: './src/index.tsx',
+	resolve: {
+		extensions: ['.ts', '.tsx', '.js'],
+		plugins: [new TsconfigPathsPlugin({ configFile: './tsconfig.json' })],
```

Comment:
I think tsconfig.json is the default one and does not need to be explicitly configured